sources/core/objects/mobs/Human.cpp

```diff
@@ -679,6 +689,12 @@ void Human::ApplySuffocationDamage(int damage)
     suffocation_damage_ = qMax(0, suffocation_damage_);
 }
 
+void Human::ApplyToxinsDamage(int damage)
+{
+    toxins_damage_ += damage;
+    toxins_damage_ = qMax(0, toxins_damage_);
```

Comment:
`std::max` instead of `qMax` will be better.
Currently Qt features are used only when they are different (and better) from std ones. If functionality is the same, then std is preferred.
Again, there are a lot of `qMax` in the old code, but it will be fixed someday.